src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+is_a: MONDO:0005583 {source="OMIA:000666"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000666"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009661 {source="OMIA:000666"} ! cross-species analog mucopolysaccharidosis type 6
+
+[Term]
+id: MONDO:1010175
+name: mucopolysaccharidosis VII, non-human animal
+def: "Mucopolysaccharidosis VII that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000667 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000667"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000667"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009662 {source="OMIA:000667"} ! cross-species analog mucopolysaccharidosis type 7
+
+[Term]
+id: MONDO:1010176
+name: Niemann-Pick disease, type C1, non-human animal
+def: "Niemann-Pick disease, type C1 that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000725 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000725"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000725"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009757 {source="OMIA:000725"} ! cross-species analog Niemann-Pick disease, type C1
+
+[Term]
+id: MONDO:1010177
+name: mucopolysaccharidosis IIIA, non-human animal
+def: "Mucopolysaccharidosis IIIA that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001309 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001309"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001309"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009655 {source="OMIA:001309"} ! cross-species analog mucopolysaccharidosis type 3A
+
+[Term]
+id: MONDO:1010178
+name: gangliosidosis, GM2, GM2A deficiency, non-human animal
+def: "Gangliosidosis, GM2, GM2A deficiency that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001427 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001427"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001427"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0010099 {source="OMIA:001427"} ! cross-species analog Tay-Sachs disease AB variant
+
+[Term]
+id: MONDO:1010179
+name: neuronal ceroid lipofuscinosis, 6, non-human animal
+def: "Neuronal ceroid lipofuscinosis, 6 that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001443 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001443"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001443"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0011144 {source="OMIA:001443"} ! cross-species analog ceroid lipofuscinosis, neuronal, 6A
+
+[Term]
+id: MONDO:1010180
+name: gangliosidosis, GM2, type II (Sandhoff or variant 0), non-human animal
```

Comment:
In human, "disease ~ type x" is based on the gene affected. Should we have only the species-specific term(s) instead of this generic term based on "general species unspecific" gene? 
Note that these species-specific diseases (based on species-specific genes) could be analogous to the human type x disease, but it is difficult to create an axiom referring to the general "species-not- specific" gene.